records_mover/db/mysql/load_options.py

```diff
@@ -69,10 +69,12 @@ def generate_load_data_sql(self,
                                filename: str,
                                schema_name: str,
                                table_name: str) -> TextClause:
+        remove_backticks_schema_name = schema_name.replace('`', '')
+        remove_backticks_table_name = table_name.replace('`', '')
         sql = f"""\
 LOAD DATA
 LOCAL INFILE :filename
-INTO TABLE {schema_name}.{table_name}
+INTO TABLE `{remove_backticks_schema_name}`.`{remove_backticks_table_name}`
```

Comment:
will this work?? if we passed a table originally containing backticks because we thought it sholud be quoted, it will end up as ``'table'`, is that equivalent to ``table`` without the single quotes?

Reply:
This is a great point, we also already have the quote schema and table method, going to try to use that

Reply:
Actually, I can't use `quote_schema_and_table` here because I'd need to pass an engine. So, while this is a terrible solution, I think it's our best one.